package.json

```diff
@@ -240,7 +240,7 @@
     "github-markdown-css": "^2.10.0",
```

Comment:
@mistic I haven't included the `"globby/fast-glob": "3.2.5"` resolution you've suggested yet, but will do before I merge this PR.

Reply:
Done.